yt/visualization/plot_modifications.py

```diff
@@ -1643,7 +1645,11 @@ def __call__(self, plot):
         y0, y1 = plot.ylim
         xx0, xx1 = plot._axes.get_xlim()
         yy0, yy1 = plot._axes.get_ylim()
-        reg = self._get_region((x0,x1), (y0,y1), plot.data.axis, data)
+        if type(self.data_source)==YTCutRegion:
+            mylog.warn("Parameter 'width' is ignored in annotate_particles if the data_source is a cut_region.")
+            self.region=self.data_source
+        else:
+            self._get_region((x0,x1), (y0,y1), plot.data.axis, data)
```

Comment:
The width is incompatible with a cut_region data_source due to issue #1933 .

Using width + a cut_region leads to plots like this one, instead of the one shown in the description of the pull request.

![projection_density_00020_z_kpc0010 00000_sink_2018_07_31_noif](https://user-images.githubusercontent.com/8629688/43645824-60860bcc-9733-11e8-8478-7e3523d5b5d2.png)

Reply:
Not necessarily, as self.region is set within _get_region. I've added it anyway for consistency.